Model/Characters/Character.cs

```diff
@@ -1,16 +1,20 @@
+using Godot;
+using Microsoft.VisualBasic;
 using Model.Entities;
 using System.Collections.Generic;
 
 namespace Model.Characters
 {
 	public class Character
 	{
+		
 		public string Name { get; set; }
 
 		// Insert sprite object later when it's developed
 		// public ??? Sprite { get; set; }
 
-		public List<Entity> EntityTeam { get; set; }
+		public List<Entity> EntityTeam { 
+			get; set; }
 
 		public bool AlreadyFought { get; set; } = false;
 	}
```

Comment:
Revert this file if nothing was really changed

Reply:
From this branch in Git, you can just run this to revert it:
git checkout master -- PathToTheFile